modules/cli/src/main/scala/scala/cli/commands/update/Update.scala

```diff
@@ -188,4 +190,41 @@ object Update extends ScalaCommand[UpdateOptions] {
 
     classesDir.contains(binRepoDir)
   }
+
+  // from https://github.com/coursier/coursier/blob/7b7c2c312aea26e850f0cd2cf15e688d0777f819/modules/cache/jvm/src/main/scala/coursier/cache/CacheUrl.scala#L489-L497
+  private def closeConn(conn: URLConnection): Unit = {
```

Comment:
NIT: we could extract this to a util object or an extension on `URLConnection`
```suggestion
  private def closeConnection(conn: URLConnection): Unit = {
```

Reply:
I'd say there's no other need to download stuff outside of coursier, so keeping this private in Update is a reasonable choice for now .